recipes/pykep/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DBUILD_SPICE=yes \
```

Comment:
`cspice` is already packaged in conda-forge. Is there a way to use that instead of using the bundled version?

Reply:
I would suggest for this 2.0 release to leave it as is. I will try to use conda spice for 2.1, but will need to modify the build system as I still need to compile the whole spice for windows wheels (PiPy)